pygmt/helpers/utils.py

```diff
@@ -119,7 +119,7 @@ def dummy_context(arg):
     yield arg
 
 
-def build_arg_string(kwargs):
+def build_arg_string(kwargs, infile=None, outfile=None):
```

Comment:
In Python, kwargs is usually placed at the last position, but here it is placed as first. Maybe we should change the ordering of the parameters here? See also @meghanrjones's comment at https://github.com/GenericMappingTools/pygmt/pull/731#discussion_r825477816 and https://peps.python.org/pep-3102 about keyword-only arguments.

Reply:
Actually, it's just a dict which has a name `kwargs`. Maybe `kwargs` is not a good name here.

Reply:
How about `kwdict`?